stream_alert/__init__.py

```diff
@@ -1,2 +1,2 @@
 """StreamAlert version."""
-__version__ = '2.3.0'
+__version__ = '3.0.0'
```

Comment:
I believe we don't upgrade version until we cut the release.

Reply:
... this is fine. this isn't being cut yet and is a WIP branch

Reply:
Yea I figured we could just bump it cuz it's going on `release-3-0-0` and we don't **_really_** use the version for anything important, if I'm not mistaken